rfcs/0000-User&HostIntegrationRFC.md

```diff
@@ -0,0 +1,254 @@
+
+<!--
+As you work on your RFC, use the "Stage N" comments to guide you in what you should focus on, for the stage you're targeting.
+Feel free to remove these comments as you go along.
+-->
+
+<!--
+Stage 0: Provide a high level summary of the premise of these changes. Briefly describe the nature, purpose, and impact of the changes. ~2-5 sentences.
+-->
+
+This proposal aims to extend existing ECS fieldset to store inventory metadata for hosts, users from external application repositories. Using ECS to store such fields will improve metadata queryring and retrieval across various use cases.
+
+Within the Security `Entity Analytics` initiatve, we refer to hosts and users as `entities`. We also refer to hosts/ users as `assets` across generic security or obserbality use cases. For simplificaiton, entities, assets will refer to host and user objects.
+
+This proposal includes the following:
+* Additional fields in the `users` object 
+* Introduces a new field set called `assets`
+<!-- * Additional fields in the `host` object --->
+
+This proposal will also facilitate storing of host and user inventory within security solution (aka. the entity store).
+
+### References:
+* [Entity Analytics Plugin Architecture for Host & User Integrations](https://github.com/elastic/beats/issues/33866#issuecomment-1165709940)
+* 0038: Extend event.kind allowed values with asset [Existing RFC](https://github.com/elastic/ecs/blob/main/rfcs/text/0038-event-kind-asset.md#0038-extend-eventkind-allowed-values-with-asset)
+* [Creation of an asset `super-schema`](https://github.com/elastic/security-team/issues/2347)
+
+<!--
+Stage 1: If the changes include field additions or modifications, please create a folder titled as the RFC number under rfcs/text/. This will be where proposed schema changes as standalone YAML files or extended example mappings and larger source documents will go as the RFC is iterated upon.
+-->
+
+<!--
+Stage X: Provide a brief explanation of why the proposal is being marked as abandoned. This is useful context for anyone revisiting this proposal or considering similar changes later on.
+-->
+
+## Fields
+
+<!--
+Stage 1: Describe at a high level how this change affects fields. Include new or updated yml field definitions for all of the essential fields in this draft. While not exhaustive, the fields documented here should be comprehensive enough to deeply evaluate the technical considerations of this change. The goal here is to validate the technical details for all essential fields and to provide a basis for adding experimental field definitions to the schema. Use GitHub code blocks with yml syntax formatting, and add them to the corresponding RFC folder.
+-->
+
+### Proposed New Fields for User object
+
+Field | Type | Example | Description
+--- | --- | --- | ---
+user.profile.id	| keyword	| 1234	| User ID from the identity datasource.
```

Comment:
What is the significance for introducing `user.profile.id` vs. using the existing `user.id` field?

Reply:
I am just getting this resolved for now. Please lmk, or reopen, if we need to reconsider.